test/xpu_api/utilities/meta/meta.trans/meta.trans.ptr/add_pointer.pass.cpp

```diff
@@ -0,0 +1,116 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#ifdef USE_ONEAPI_STD
+namespace s = oneapi_cpp_ns;
+#else
```

Comment:
What is this change for? I did not find any mentions of `USE_ONEAPI_STD` and `oneapi_cpp_ns` in oneDPL repo and Google:)

Reply:
Fixed: it's not actual code from source test.